CMakeLists.txt

```diff
@@ -187,13 +187,17 @@ set(XPYT_SEP ":") # TODO: use ";" on windows
 
 string(CONCAT XEUS_SYSPATH
  "" ${XPYT_SEP}
+ # System sys.path
+ "/usr/lib/python${PY_VERSION_MAJOR}${PY_VERSION_MINOR}.zip" ${XPYT_SEP}
+ "/usr/lib/python${PY_VERSION_MAJOR}.${PY_VERSION_MINOR}" ${XPYT_SEP}
+ "/usr/lib/python${PY_VERSION_MAJOR}.${PY_VERSION_MINOR}/lib-dynload" ${XPYT_SEP}
```

Comment:
We cannot add the "System sys.path" in that way. In fact, it should be taken care of by the other cases below, because `{PREFIX}` is replaced by the actual installation prefix (`/usr/local` on Fedora).

Reply:
The reason why it does not work in your case is that each Linux distribution patches CPython in a different way.

To support the case of Debian, which does the opposite of Fedora, we need to add `dist-packages` and `{PREFIX}/local/...`.

I wish Linux distributions only packaged stock CPython instead of patching.

Reply:
I will post a general description of the issue in a separate issue.

Reply:
Hmmm - I'm on Arch, but I'm also using a virtualenv. Hard-coding like this means that the standard libraries aren't able to be loaded

Reply:
Yes, but depending on your Python installation, the root environment packages are not always in the standard library.

Reply:
Is there another way to get this information at installation, rather than at packaging time?

Reply:
Probably, I am right in the middle of opening a more thorough description of the issue.

Reply:
Good stuff :)